libs/application-generic/package.json

```diff
@@ -5,11 +5,7 @@
   "main": "build/main/index.js",
   "typings": "build/main/index.d.ts",
   "module": "build/module/index.js",
-  "private": false,
-  "repository": {
-    "url": "https://github.com/novuhq/novu",
-    "directory": "packages/nest"
-  },
+  "private": true,
```

Comment:
All the other files are moved under the `libs` folder. Package.json was changed to make this package public.